data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml

```diff
@@ -65,20 +65,21 @@
   <listBibl xml:lang="lat">
    <biblStruct>
    <monogr>
-    <title>Philonis Alexandri Opera Quae Supersunt</title>
+    <title xml:lang="lat">Philonis Alexandri Opera Quae Supersunt</title>
     <editor>
     <persName>
      <name xml:lang="lat">Leopoldus Cohn</name>
     </persName>
     </editor>
     <author ref="urn:tmp">Philo Judaeus</author>
     <imprint>
-    <publisher>Typis et Impensis Georgi Reimeri</publisher>
-    <pubPlace>Berolini</pubPlace>
+    <publisher xml:lang="lat">Typis et Impensis Georgi Reimeri</publisher>
+    <pubPlace xml:lang="lat">Berolini</pubPlace>
     <date>1896</date>
     </imprint>
     <biblScope unit="volume">vol. i</biblScope>
    </monogr>
+     <ref target="https://babel.hathitrust.org/cgi/pt?id=hvd.32044054757893;view=1up;seq=126">Internet Archive</ref>
```

Comment:
Since these scans actually come from Hathi Trust and not the Internet Archive, please change the text of the elements to "Hathi Trust" (I think you need to change this in all of the files).